package.json

```diff
@@ -8,7 +8,8 @@
     "build": "node bin/build.js",
     "start": "npm run clean && npm run build",
     "version": "npm run build && git add -A dist dist/* jquery-src bin",
-    "test": "karma start test/karma.conf.js"
+    "test": "karma start test/karma.conf.js",
+    "test:jasmine": "jasmine test/test.js"
```

Comment:
(update after talking with @n-riesco - we'll keep karma available since this package is generally targeted at browsers, but jasmine is a nice addition for rapid iteration)